docs/topics/articles.md

```diff
@@ -0,0 +1,184 @@
+# `Article`s and `ListingPage`s
+
+`Article`s are publishable content types that live in collections. Normally
+articles are mounted under a parent's URL.
+
+An example of an article is a Press Release, which would be mounted under a
+Press Release Listing Page. The Listing Page has the URL "/press" and thus all
+the Press Releases have the URL "/press/<slug>".
+
+## `ArticleBase`
+
+The `icekit.articles.models.ArticleBase` inherits `title`, `slug` and
+publishing fields, and each `ArticleBase` subclass implements some key
+functionality:
+
+1. **Links to parents.**
+In order for an Article to know its URL (which is based on a parent's URL), it
+should define a `parent` link, which is normally a link to a `ListingPage`.
```

Comment:
Insisting on an FK might be OK, but I'd rather not make people choose 'press page' for every  press release. At the moment I think we cross that bridge when we come to it, as the migration is likely to be straightforward.

FWIW, I'm implementing `parent` like this:
```    parent = models.ForeignKey(
        "ArticleCategoryPage",
        limit_choices_to={'publishing_is_draft': True},
        on_delete=models.PROTECT,
    )
```